dev/com.ibm.etools.maven.liberty.integration/src/com/ibm/etools/maven/liberty/integration/manager/internal/LibertyManager.java

```diff
@@ -44,4 +47,15 @@ protected boolean handleGenerationPrompt(String projectName) {
         return UIHelper.handleGenerationPrompt(LibertyMavenConstants.PROMPT_PREFERENCE, NLS.bind(Messages.generationPromptMsg, projectName));
     }
 
+    /** {@inheritDoc} */
+    @Override
+    protected boolean isApplicableProjectToScan(IProject project) {
+        try {
+            return project.hasNature(LibertyMavenConstants.MAVEN_PROJECT_NATURE);
+        } catch (CoreException e) {
+            Trace.trace(Trace.INFO, "Error getting project nature for" + project.getName(), e);
```

Comment:
Trace levels other than ERROR should be surrounded by if (Trace.ENABLED).  This looks more like a warning than an info as well.